patientsearch.env.default

```diff
@@ -7,3 +7,7 @@ SERVER_NAME=localtest.me:8000
 
 # generate long random alphanumeric string
 SECRET_KEY=
+
+# log server URL and TOKEN (JWT, see https://github.com/uwcirg/logserver#access-via-jwt)
+LOGSERVER_TOKEN=
+LOGSERVER_URL=
```

Comment:
I try to comment out non-required vars, so it's not as overwhelming when configuring a new system
```suggestion
# LOGSERVER_TOKEN=
# LOGSERVER_URL=
```